src/boot/ZulipAsyncStorage.js

```diff
@@ -58,6 +58,23 @@ export default class ZulipAsyncStorage {
     );
   }
 
+  static async multiSet(
```

Comment:
Is it reasonable to test this? If not, why not?

Reply:
Replied in chat, [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/.60jest-expo.2Fios.60.20and.20.60jest-expo.2Fandroid.60/near/1169139).

Reply:
#4700